shared/search/user-search/render.native.js

```diff
@@ -42,13 +45,13 @@ function KeybaseExtraInfo ({username, fullName, isFollowing, searchText}) {
   )
 }
 
-// TODO(MM) use serviceAvatar
-function ExternalExtraInfo ({serviceUsername, fullNameOnService, icon, searchText}) {
+function ExternalExtraInfo ({fullNameOnService, icon, serviceAvatar, serviceUsername, searchText}) {
```

Comment:
If we have the externalHint and external variants of extra info, we'd have two simple functions instead of trying to show an icon or avatar.